treeherder/etl/job_loader.py

```diff
@@ -58,7 +60,10 @@ def process_job_list(self, all_jobs_list):
                             logger.warn("Skipping job due to bad attribute",
                                         exc_info=1)
 
-                jobs_model.store_job_data(storeable_job_list)
+                try:
+                    jobs_model.store_job_data(storeable_job_list)
+                except(DatasetNotFoundError):
```

Comment:
s/except(DatasetNotFoundError)/except DatasetNotFoundError/

I'm also surprised the `clean_revision()` call above doesn't raise `DatasetNotFoundError` too?